reference/configuration/framework.rst

```diff
@@ -854,6 +852,42 @@ If you're using forms, but want to avoid starting your session (e.g. using
 forms in an API-only website), ``csrf_protection`` will need to be set to
 ``false``.
 
+stateless_token_ids
+...................
+
+**type**: ``array`` **default**: ``[]``
+
+The list of CSRF token ids that will use stateless CSRF protection.
+
+.. versionadded:: 7.2
+
+    This option was added in Symfony 7.2 to aid in configuring stateless CSRF protection.
+
+check_header
+............
+
+**type**: ``integer|bool`` **default**: ``false``
```

Comment:
Given https://github.com/symfony/symfony-docs/pull/20962#issuecomment-2882692632 and https://github.com/symfony/symfony-docs/commit/90e7bf98639ce9bbd6b60e5f4a7df47d5dc87e10, I think it must be: 
```suggestion
**type**: ``integer`` or ``bool`` **default**: ``false``
```

Reply:
Also, in this case, the `|` is inside a code literal, so it should not cause any issue AFAICT, as code literals are disabling any special markup parsing inside them.
So #20962 indeed fix invalid markup, but the markup replaced in https://github.com/symfony/symfony-docs/commit/90e7bf98639ce9bbd6b60e5f4a7df47d5dc87e10 was not invalid (it can be considered as improving consistency in the doc)